### PR TITLE
[ADDomainService]Changed the input method of certificate from base64 string to file path

### PR DESCRIPTION
**Pull request by BethanyZhou**
## Description
Changelog: Changed the input method of certificate from base64 string to file path

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/ADDomainServices/README.md [hunk 1]
```diff
@@ -58,6 +59,9 @@ directive:
   - from: swagger-document
     where: $.definitions..trustPassword
     transform: $.format = "password"
+  - from: swagger-document
+    where: $.definitions..pfxCertificate
```
Comment (VeryEarly) [inline src/ADDomainServices/README.md:63]: should it be
$.definitions.LdapsSettings.pfxCertificate ?

**Comment by VeryEarly** [comment]
seems like two cmdlet are renamed without alias, can you please confirm

File: src/ADDomainServices/README.md (hunk 1 above)
Comment (BethanyZhou) [reply src/ADDomainServices/README.md:63]: Yes. $.definitions..pfxCertificate means $.definitions.*.pfxCertificate. It works as well.

**Comment by BethanyZhou** [comment]
Yes. Renaming these two cmdlets without alias is expected.  Renamed 'New-AzADDomainServiceForestTrust'->'New-AzADDomainServiceForestTrustObject and 'New-AzADDomainServiceReplicaSet' -> 'New-AzADDomainServiceReplicaSetObject' to align with our naming convention. And Az.ADDomainService is under preview. I guess these changes should be fine.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**